### PR TITLE
flush cache during dev

### PR DESCRIPTION
**Pull request by kyscott18**

### CODE REVIEW

File: packages/core/src/bin/utils/run.ts [hunk 1]
```diff
@@ -175,7 +175,11 @@ export async function run({
       // Persist the indexing store to the db if it is too full. The `finalized`
       // checkpoint is used as a mutex. Any rows in the reorg table that may
       // have been written because of raw sql access are deleted.
-      if (historicalIndexingStore.isCacheFull() && events.length > 0) {
+      if (
+        (historicalIndexingStore.isCacheFull() ||
+          common.options.command === "dev") &&
```
Comment (kyscott18) [inline packages/core/src/bin/utils/run.ts:180]: @typedarray Should we using a different algorithm than this?

Comment (typedarray) [reply packages/core/src/bin/utils/run.ts:180]: Approx how often will this run?

Comment (kyscott18) [reply packages/core/src/bin/utils/run.ts:180]: every 10k events

**Review by typedarray — APPROVED** [review]
LGTM

**Pull request merged**